sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,92 @@
+(28912,5,0,'I can''t keep this barrier up much longer... Where is that coward?',12,0,100,0,0,0,29208,0,'Koltira Deathweaver'),
+(28912,6,0,'The High Inquisitor comes! Be ready, death knight! Do not let him draw you out of the protective bounds of my anti-magic field! Kill him and take his head!',12,0,100,0,0,0,29210,0,'Koltira Deathweaver'),
+(28912,7,0,'Stay in the anti-magic field! Make them come to you!',12,0,25,0,0,0,29225,0,'Koltira Deathweaver'),
+(28912,7,1,'Stand your ground! Use my anti-magic barrier to dampen their spells!',12,0,25,0,0,0,29224,0,'Koltira Deathweaver'),
+(28912,8,0,'The death of the High Inquisitor of New Avalon will not go unnoticed. You need to get out of here at once! Go, before more of them show up. I''ll be fine on my own.',12,0,100,1,0,0,29239,0,'Koltira Deathweaver'),
+(28912,9,0,'I''ll draw their fire, you make your escape behind me.',12,0,100,1,0,0,29240,0,'Koltira Deathweaver'),
+(28912,10,0,'Your High Inquisitor is nothing more than a pile of meat, Crusaders! There are none beyond the grasp of the Scourge!',14,0,100,22,0,0,29241,0,'Koltira Deathweaver');
+
+DELETE FROM `creature_summon_groups` WHERE `summonerId`=28912;
+INSERT INTO `creature_summon_groups` (`summonerId`,`summonerType`,`groupId`,`entry`,`position_x`,`position_y`,`position_z`,`orientation`,`summonType`,`summonTime`) VALUES
+(28912,0,0,29007,1640.672,-6032.053,134.8221,4.679538,6,5000),
+(28912,0,0,29007,1641.005,-6031.893,134.8221,0.4014257,6,5000),
+(28912,0,0,29007,1639.705,-6031.737,134.8221,2.443461,6,5000),
+(28912,0,1,29007,1640.404,-6032.289,134.7387,4.517092,6,5000),
+(28912,0,1,29007,1641.73,-6030.751,134.8221,6.143559,6,5000),
+(28912,0,1,29007,1639.466,-6030.404,134.8221,4.502949,6,5000),
+(28912,0,2,29007,1640.286,-6033.075,134.6049,4.679538,6,5000),
+(28912,0,2,29007,1642.04,-6030.384,134.8221,1.37881,6,5000),
+(28912,0,2,29007,1640.116,-6029.782,134.8221,5.707227,6,5000),
+(28912,0,2,29007,1640.995,-6029.803,134.8221,1.605703,6,5000),
+(28912,0,3,29001,1640.766,-6032.984,134.5805,4.596632,5,0);
+
+DELETE FROM `waypoint_data` WHERE `id`=28912;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `delay`, `action`, `orientation`, `action_chance`, `move_type`) VALUES
+(28912,1,1640.65,-6032.49,134.739,0,0,0,100,1),
+(28912,2,1639.1,-6028.1,134.739,0,0,0,100,1),
+(28912,3,1628.37,-6026.34,134.739,0,0,0,100,1),
+(28912,4,1625.79,-6021.81,134.739,0,0,0,100,1),
+(28912,5,1627.06,-6016.95,134.739,0,0,0,100,1),
+(28912,6,1639.85,-6015.64,134.74,0,0,0,100,1),
+(28912,7,1653.376,-5971.874,132.2567,0,0,0,100,1),
+(28912,8,1653.61,-5927.44,121.345,0,0,0,100,1),
+(28912,9,1656.07,-5905.5,116.21,0,0,0,100,1),
+(28912,10,1681.736,-5888.621,116.257,0,1188,0,100,1);
+UPDATE `creature_addon` SET `bytes1`=0,`auras`='' WHERE `guid`=130354;
+
+DELETE FROM `creature_equip_template` WHERE `CreatureID`=25446;
```

Comment:
Wrong id?

Reply:
Fair point. "Warsong Captain" seems unlikely.

Reply:
Yes wrong id, but also why removing the equipment from the DB?

Reply:
Then
```sql
DELETE FROM `creature_equip_template` WHERE `CreatureID`=28912 AND `ID`=1;

Reply:
The follow-up insert contains `28912, 1, 38633, 0,0,0` which uses 28912=Koltira Deathweaver + 38633=Greataxe of the Ebon Blade.

Reply:
I thought the equipment was already in the DB.